runners/spark/src/main/java/org/apache/beam/runners/spark/translation/GroupCombineFunctions.java

```diff
@@ -203,4 +203,26 @@
 
     return accumulatedBytes.mapToPair(CoderHelpers.fromByteFunction(keyCoder, iterAccumCoder));
   }
+
+  /**
+   * An implementation of
+   * {@link org.apache.beam.sdk.util.Reshuffle} for the Spark runner.
+   */
+  public static <K, V> JavaRDD<WindowedValue<KV<K, V>>> reshuffle(
+      JavaRDD<WindowedValue<KV<K, V>>> rdd,
+      Coder<K> keyCoder,
+      WindowedValueCoder<V> wvCoder) {
+
+    // Use coders to convert objects in the PCollection to byte arrays, so they
+    // can be transferred over the network for the shuffle.
+    return rdd
```

Comment:
I'm assuming you're "Reifying" to preserve the window before transforming to a pair ? I know that this is how Reshuffle is implemented in Beam, but I wonder if the Spark implementation really cares about it ?
Isn't it enough to perform: 
`JavaRDD<WindowedValue<T>>` -> `JavaRDD<byte[]>` (using a `FullWindowedValueCoder`) -> repartition -> and back to deserialized representation ?